include/boost/gil/image_processing/numeric.hpp

```diff
@@ -50,6 +50,24 @@ inline double lanczos(double x, std::ptrdiff_t a)
     return 0;
 }
 
+inline void compute_tensor_entries(
+    boost::gil::gray16_view_t dx,
+    boost::gil::gray16_view_t dy,
```

Comment:
Why namespace qualification here but earlier not? I don't think this is nessecary.

Reply:
This will be addressed as part of #352